app/assets/javascripts/components/ecnumbers.js

```diff
@@ -0,0 +1,260 @@
+import PriorityQueue from "../utilities/PriorityQueue.js";
+import {postJSON} from "../utils.js";
+/**
+ * @typedef {Object} FACounts
+ * @property {number} value count
+ * @property {string} name  The name of the GO/EC number
+ * @property {string} code  The code of the GO/EC number
+ */
+
+// const for private methods
+const addMissing = Symbol("[addMissing]");
```

Comment:
to create "private" static methods, I'll make them simple static methods. In retrospect symbols were not needed

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13732) by @beardhatcode on Mon Feb 19 2018 at 15:20._